website/content/en/preview/upgrading/v1-migration.md

```diff
@@ -0,0 +1,246 @@
+---
+title: "v1 Migration"
+linkTitle: "v1 Migration"
+weight: 30
+description: >
+  Upgrade information for migrating to v1
+---
+
+This migration guide is designed to help you migrate Karpenter from a pre-v1 version (v1beta1) to Karpenter v1.
+Use this document as a reference to the changes that were introduced in this release and as a guide to how you need to update the manifests and other Karpenter objects you created in previous Karpenter releases.
+
+Before you begin upgrading to `1.0.0`, you should know that:
```

Comment:
Since this is talking about pre-v1 actions, should we ref out to the section where you describe some of pre-v1 actions in the changelog below?

Reply:
I added this sentence below the bullet list. Should I remove the bullet list as well? 
`See the [Changelog]({{<ref "#changelog" >}}) for details about actions you should take before upgrading to v1.0 or v1.1.`